lib/active_fedora/file/attributes.rb

```diff
@@ -72,8 +72,8 @@ def default_mime_type
     end
 
     def fetch_mime_type
-      return default_mime_type if new_record?
-      ldp_source.head.content_type
+      return default_mime_type if new_record? && metadata.mime_type.blank?
+      metadata.mime_type.first
```

Comment:
I think it's odd for a file to have more than one mime-type, and changing it would likely cause an issue in hydra-derivatives.  The mime_type has always returned a scalar value.